addons/transifex/models/transifex_code_translation.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from configparser import ConfigParser
+from os.path import join as opj
+import os
```

Comment:
```suggestion
from os import path import isfile, join as opj
```
to avoid having a global `os` import